File: Angular/src/app/models/Requirement.ts
export class Requirement{

    constructor(minAge: number, maxAge: number, gender: string, reputation: number){
        this.minAge = minAge;
        this.maxAge = maxAge;
        this.gender = gender;
        this.reputation = reputation;
    }

    minAge: number;
    maxAge: number;
    gender: string;
    reputation: number;
}

File: API/src/main/java/com/laura/api/model/Requirement.java
package com.laura.api.model;

import javax.persistence.Embeddable;

@Embeddable
public class Requirement {

    private int minAge;
    private int maxAge;
    private String gender;  
    private float reputation;

    public int getMinAge() {
        return minAge;
    }

    public void setMinAge(int minAge) {
        this.minAge = minAge;
    }

    public int getMaxAge() {
        return maxAge;
    }

    public void setMaxAge(int maxAge) {
        this.maxAge = maxAge;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public float getReputation() {
        return reputation;
    }

    public void setReputation(float reputation) {
        this.reputation = reputation;
    }

}
